prebuilt-ui/basic-embed/components/PrebuiltCall.js

```diff
@@ -0,0 +1,254 @@
+
+  const content = useMemo(() => {
+    switch (demoState) {
+      case 'home':
+        return (
+          <Card>
+            <CardHeader>
+              Start demo with a new unique room, or paste in your own room URL.
+            </CardHeader>
+            <CardBody>
+              {!props.configured && (
+                <Well variant="error">
+                  You must configure env variables to create rooms (see README
+                  instructions).
+                </Well>
+              )}
+              {isError && (
+                <Well variant="error">
+                  Error creating the room. Please try again.
+                </Well>
+              )}
+              <Button onClick={() => createRoom()} disabled={roomValidity}>
+                Create room and start
+              </Button>
+              <Field label="Or enter room to join" className="roomField">
+                <TextInput
+                  ref={roomURLRef}
+                  type="text"
+                  placeholder="Enter room URL..."
+                  pattern="^(https:\/\/)?[\w.-]+(\.(daily\.(co)))+[\/\/]+[\w.-]+$"
+                  onChange={handleRoomInput}
+                />
+              </Field>
+              <CardFooter>
+                <Button
+                  onClick={() => submitJoinRoom()}
+                  disabled={!roomValidity}
+                >
+                  Join room
+                </Button>
+              </CardFooter>
+            </CardBody>
+          </Card>
+        );
+      case 'call':
+        return (
+          <>
+            <div ref={iframeRef} className="call" />
+            <Card>
+              <CardHeader>Copy and share the URL to invite others.</CardHeader>
+              <CardBody>
+                <label for="copy-url"></label>
+                <TextInput
+                  type="text"
+                  class="url-input"
+                  id="copy-url"
+                  placeholder="Copy this room URL"
+                  value={roomURL}
+                  pattern="^(https:\/\/)?[\w.-]+(\.(daily\.(co)))+[\/\/]+[\w.-]+$"
+                />
+                <Button onClick={handleCopyClick}>
+                  {linkCopied ? 'Copied!' : `Copy room URL`}
+                </Button>
+                {exp && (
+                  <h3>
+                    This room expires in:{' '}
+                    <span className="countdown">{secs}</span>
+                  </h3>
+                )}
+              </CardBody>
+            </Card>
+          </>
+        );
+    }
+  }, [demoState, roomValidity, roomURLRef, exp, secs]);
+
+  return (
+    <div className="container">
+      <Header />
+      {content}
+      <style jsx>{`
+        .container {
+          display: flex;
+          align-items: center;
+          justify-content: space-around;
+          width: 100%;
+          height: 100%;
+          position: absolute;
+        }
+
+        .container :global(.call) {
+          height: 70%;
```

Comment:
I think you want `width:100%` instead of the height property to maintain the aspect ratio set in the call options

Reply:
I changed this, and added a wrapper "inCall" class around the card and call to keep the content more centered. I think this is _better_, but there is probably an even better way to do this.

Reply:
I tested it locally removing the height property and replacing it with `width:100%` and it maintained the aspect ratio you have declared and filled the space, so I think you don't need a wrapper too. I can add a screenshot of what I mean tomorrow.

Reply:
Agreed! I've got another pass up that's something better, so I'll resolve this for now since the next round is ready for review.